Author: Eagle Burkut <eagle.burkut@gmail.com>
Bug: http://sourceware.org/bugzilla/show_bug.cgi?id=13759

Index: langpack-locales/locales/ug_CN
===================================================================
--- langpack-locales.orig/locales/ug_CN	2011-11-16 08:25:08.000000000 +0100
+++ langpack-locales/locales/ug_CN	2012-02-27 11:13:50.904504537 +0100
@@ -1,251 +1,219 @@
 escape_char /
 comment_char %
 %
-% Uyghur language locale for China
+% Uyghur locale for China
 % Source:
-% Contact: Pablo Saratxaga
-% Email: pablo@mandriva.com
-% Language: ug
+% Contact:   Eagle Burkut
+% Email:     eagle.burkut@gmail.com
+% Language:  ug
 % Territory: CN
-% Revision: 0.1
-% Date: 2005-11-08
-% Users: general
-% Charset: UTF-8
-% Distribution and use is free, also
-% for commercial purposes.
+% Revision:  2.00
+% Date:      2012-02-26
+% Users:     general
+% Charset:   UTF-8
+% Distribution and use is free,
+% also for commercial purposes.
 
 LC_IDENTIFICATION
-title      "Uyghur locale for China"
-source     ""
-address    ""
-contact    ""
-email      "pablo@mandriva.com"
-tel        ""
-fax        ""
-language   "Uyghur"
-territory  "China"
-revision   "0.1"
-date       "2005-11-08"
+title     "Uyghur locale for China"
+source    ""
+address   ""
+contact   "Eagle Burkut"
+email     "eagle.burkut@gmail.com"
+tel       ""
+fax       ""
+language  "Uyghur"
+territory "China"
+revision  "2.00"
+date      "2011-02-26"
 %
-category  "ug_CN:2000";LC_IDENTIFICATION
-category  "ug_CN:2000";LC_CTYPE
-category  "ug_CN:2000";LC_COLLATE
-category  "ug_CN:2000";LC_TIME
-category  "ug_CN:2000";LC_NUMERIC
-category  "ug_CN:2000";LC_MONETARY
-category  "ug_CN:2000";LC_MESSAGES
-category  "ug_CN:2000";LC_PAPER
-category  "ug_CN:2000";LC_NAME
-category  "ug_CN:2000";LC_ADDRESS
-category  "ug_CN:2000";LC_TELEPHONE
+category "ug_CN:2000";LC_IDENTIFICATION
+category "ug_CN:2000";LC_CTYPE
+category "ug_CN:2000";LC_COLLATE
+category "ug_CN:2000";LC_TIME
+category "ug_CN:2000";LC_NUMERIC
+category "ug_CN:2000";LC_MONETARY
+category "ug_CN:2000";LC_MESSAGES
+category "ug_CN:2000";LC_PAPER
+category "ug_CN:2000";LC_NAME
+category "ug_CN:2000";LC_ADDRESS
+category "ug_CN:2000";LC_TELEPHONE
 
 END LC_IDENTIFICATION
 
 LC_COLLATE
-
-% letters list and sort order from
+%
 % http://en.wikipedia.org/wiki/Uyghur_language
-% sorting order guessed for old latin writting
 %
-% old latin writting is based in pinyin, with some extra letters:
-% <schwa>, <z with tail>, <gh>, <k with tail>, <h with tail>,
-% <barred o> (they seem inspired from cyrillic)
-% tail letters can be written with base letter + combining tail U+0321
-% there are also reserved codepoints U+2C67-U+2C6C for them in
-% unicode 5.0 proposal.
-% U+0224, U+0225 are also similar to ztail and are sorted the same.
+% CODE - UNICODE CODE POINT
+% UL   - UYGHUR LATIN
+% IN?  - IN ISO 14651_T1
 %
-% new arabic writting uses some extra letters too.
-% all vowels are noted, and in beginning of the word there is a
-% "yeh with hamza" (U+0626) in the front; should it be ignored
-% in sorting?
+% NO CODE UL ENGLISH NAME         AR IN?
+% ======================================
+%  1 0627 a  alef                 ا
+%  2 06d5 e  ae                   ە  ???
+%  3 0628 b  beh                  ب 
+%  4 067e p  peh                  پ
+%  5 062a t  teh                  ت
+%  6 062c j  jeem                 ج
+%  7 0686 ch tcheh                چ
+%  8 062e x  khah                 خ
+%  9 062f d  dal                  د
+% 10 0631 r  reh                  ر
+% 11 0632 z  zain                 ز
+% 12 0698 zh jeh                  ژ
+% 13 0633 s  seen                 س
+% 14 0634 sh sheen                ش
+% 15 063a gh ghain                غ
+% 16 0641 f  feh                  ف
+% 17 0642 q  qaf                  ق
+% 18 0643 k  kaf                  ك
+% 19 06af g  gaf                  گ
+% 20 06ad ng ng                   ڭ ???
+% 21 0644 l  lam                  ل
+% 22 0645 m  meem                 م
+% 23 0646 n  noon                 ن
+% 24 06be h  heh doachashmee      ھ ???
+% 25 0648 o  waw                  و
+% 26 06c7 u  u                    ۇ ???
+% 27 06c6 ö  oe                   ۆ ???
+% 28 06c8 ü  yu                   ۈ ???
+% 29 06cb w  ve                   ۋ ???
+% 30 06d0 é  e                    ې ???
+% 31 0649 i  alef maksura         ى
+% 32 064a y  yeh                  ي ???
+% -- 0626 '  yeh with hamza above ئ ???
 %
-% arabic     old latin       turkic (from azeri, which has same phonemes)
-% ========== =============== ======
-% alef  0627 a               a
-% ae    06d5 schwa 018F,0259 ä
-% beh   0628 b               b
-% peh   067e p               p
-% teh   062a t               t
-% jeem  062c j               c <
-% tcheh 0686 q               ç <
-% khah  062e h               x <
-% dal   062f d               d
-% reh   0631 r               r
-% zain  0632 z               z
-% jeh   0698 ztail 0224,0225 j <
-% seen  0633 s               s
-% sheen 0634 x               ş
-% ghain 063a gh 01a2,01a3    ğ <
-% feh   0641 f               f
-% qaf   0642 ktail           q <
-% kaf   0643 k               k
-% ng    06ad ng              ñ <
-% gaf   06af g               g
-% lam   0644 l               l
-% meem  0645 m               m
-% noon  0646 n               n
-% heh   0647 htail           h <
-% waw   0648 o               o
-% u     06c7 u               u
-% oe    06c6 obar 019f,0275  ö <
-% yu    06c8 ü               ü
-% ve    06cb v               v
-% e     06d0 e               e
-% f.yeh 06cc i               i      (or alef maqsura 0649?)
-% yeh   064a y               y
-% (<): differences between old latin and turkic writting
-
 copy "iso14651_t1"
 
-collating-symbol  <schwa>
-collating-symbol  <gh>
-collating-symbol  <ng>
-collating-symbol  <ccedilla>
-collating-symbol  <scedilla>
-collating-symbol  <htail>
-collating-symbol  <ktail>
-collating-symbol  <ztail>
-collating-symbol  <obar>
-collating-symbol  <udiaeresis>
-
-collating-element <h,> from "<U0068><U0321>"
-collating-element <H,> from "<U0048><U0321>"
-collating-element <k,> from "<U006B><U0321>"
-collating-element <K,> from "<U004B><U0321>"
-collating-element <z,> from "<U007A><U0321>"
-collating-element <Z,> from "<U005A><U0321>"
-
-collating-symbol  <ar_ae>
-collating-symbol  <ar_ng>
-collating-symbol  <ar_u>
-collating-symbol  <ar_oe>
-collating-symbol  <ar_yu>
-collating-symbol  <ar_ve>
-collating-symbol  <ar_e>
-collating-symbol  <ar_i>
-collating-symbol  <ar_y>
-
-% latin
-reorder-after <a>
-<schwa>
-reorder-after <c>
-<ccedilla>
-reorder-after <g>
-<gh>
-reorder-after <h>
-<htail>
-reorder-after <k>
-<ktail>
-reorder-after <n>
-<ng>
-reorder-after <o>
-<obar>
-reorder-after <s>
-<scedilla>
-reorder-after <u>
-<udiaeresis>
-reorder-after <z>
-<ztail>
-
-reorder-after <U0061>
-<U0259>   <schwa>;<PCL>;<MIN>;IGNORE
-<U00E4>   <schwa>;<REU>;<MIN>;IGNORE
-reorder-after <U0041>
-<U018F>   <schwa>;<PCL>;<CAP>;IGNORE
-<U00C4>   <schwa>;<REU>;<CAP>;IGNORE
-
-reorder-after <U0063>
-<U00E7> <ccedilla>;<CDI>;<MIN>;IGNORE
-reorder-after <U0043>
-<U00C7> <ccedilla>;<CDI>;<CAP>;IGNORE
-
-reorder-after <U0067>
-<U01A3>   <gh>;<PCL>;<MIN>;IGNORE
-<U011F>   <gh>;<BRE>;<MIN>;IGNORE
-reorder-after <U0047>
-<U01A2>   <gh>;<PCL>;<CAP>;IGNORE
-<U011E>   <gh>;<BRE>;<CAP>;IGNORE
-
-reorder-after <U0068>
-<U2C68> <htail>;<PCL>;<MIN>;IGNORE
-<h,>    <htail>;<PCL>;<MIN>;IGNORE
-reorder-after <U0048>
-<U2C67> <htail>;<PCL>;<CAP>;IGNORE
-<H,>    <htail>;<PCL>;<CAP>;IGNORE
-
-reorder-after <U006B>
-<U2C6A> <ktail>;<PCL>;<MIN>;IGNORE
-<k,>    <ktail>;<PCL>;<MIN>;IGNORE
-reorder-after <U004B>
-<U2C69> <ktail>;<PCL>;<CAP>;IGNORE
-<K,>    <ktail>;<PCL>;<CAP>;IGNORE
-
-reorder-after <U006E>
-<U00F1> <ng>;<TIL>;<MIN>;IGNORE
-reorder-after <U004E>
-<U00D1> <ng>;<TIL>;<CAP>;IGNORE
-
-reorder-after <U006F>
-<U0275> <obar>;<PCL>;<MIN>;IGNORE
-<U00F6> <obar>;<REU>;<MIN>;IGNORE
-reorder-after <U004F>
-<U019F> <obar>;<PCL>;<CAP>;IGNORE
-<U00D6> <obar>;<REU>;<CAP>;IGNORE
-
-reorder-after <U0073>
-<U015F> <scedilla>;<CDI>;<MIN>;IGNORE
-reorder-after <U0053>
-<U015E> <scedilla>;<CDI>;<CAP>;IGNORE
-
-reorder-after <U0075>
-<U00FC> <udiaeresis>;<REU>;<MIN>;IGNORE
-reorder-after <U0055>
-<U00DC> <udiaeresis>;<REU>;<CAP>;IGNORE
-
-reorder-after <U007A>
-<U2C6C> <ztail>;<PCL>;<MIN>;IGNORE
-<z,>    <ztail>;<PCL>;<MIN>;IGNORE
-<U0225> <ztail>;<PCL>;<MIN>;IGNORE
-reorder-after <U005A>
-<U2C6B> <ztail>;<PCL>;<CAP>;IGNORE
-<Z,>    <ztail>;<PCL>;<CAP>;IGNORE
-<U0224> <ztail>;<PCL>;<CAP>;IGNORE
+% <yeh with hamza above> + <alef> = ئا
+collating-symbol  <ARA_ywha_alef>
+
+% <yeh with hamza above> + <ae> = ئە
+collating-symbol  <ARA_ywha_ae>
+
+% <ae> = ە
+collating-symbol  <ARA_ae>
+
+% <ng> = ڭ
+collating-symbol  <ARA_ng>
+
+% <heh doachashmee> = ھ
+collating-symbol  <ARA_hehd>
+
+% <yeh with hamza above> + <waw> = ئو
+collating-symbol  <ARA_ywha_waw>
+
+% <yeh with hamza above> + <u> = ئۇ
+collating-symbol  <ARA_ywha_u>
+
+% <u> = ۇ
+collating-symbol  <ARA_u>
+
+% <yeh with hamza above> + <oe> = ئۆ
+collating-symbol  <ARA_ywha_oe>
+
+% <oe> = ۆ
+collating-symbol  <ARA_oe>
+
+% <yeh with hamza above> + <yu> = ئۈ
+collating-symbol  <ARA_ywha_yu>
+
+% <yu> = ۈ
+collating-symbol  <ARA_yu>
+
+% <ve> = ۋ
+collating-symbol  <ARA_ve>
+
+% <yeh with hamza above> + <e> = ئې
+collating-symbol  <ARA_ywha_e>
+
+% <e> = ې
+collating-symbol  <ARA_e>
+
+% <yeh with hamza above> + <alef maksura> = ئى
+collating-symbol  <ARA_ywha_alma>
+
+% <yeh> = ي
+collating-symbol  <ARA_yeh>
+
+% <yeh with hamza above> = ئ
+collating-symbol  <ARA_ywha>
+
+collating-element <ara_h_alef> from "<U0626><U0627>" % ئا
+collating-element <ara_h_ae>   from "<U0626><U06D5>" % ئە
+collating-element <ara_h_waw>  from "<U0626><U0648>" % ئو
+collating-element <ara_h_u>    from "<U0626><U06C7>" % ئۇ
+collating-element <ara_h_oe>   from "<U0626><U06C6>" % ئۆ
+collating-element <ara_h_yu>   from "<U0626><U06C8>" % ئۈ
+collating-element <ara_h_e>    from "<U0626><U06D0>" % ئې 
+collating-element <ara_h_alma> from "<U0626><U0649>" % ئى
 
-% arabic
 reorder-after <alef>
-<ar_ae>
-reorder-after <keheh>
-<ar_ng>
+<ARA_ywha_alef>
+<alef>
+<ARA_ywha_ae>
+<ARA_ae>
+
+reorder-after <gaf>
+<ARA_ng>
+
+reorder-after <noon>
+<ARA_hehd>
+
 reorder-after <waw>
-<ar_u>
-<ar_oe>
-<ar_yu>
-<ar_ve>
-<ar_e>
-<ar_i>
-<ar_y>
-
-reorder-after <U0627>
-<U06D5> <ar_ae>;<BAS>;<MIN>;IGNORE
-reorder-after <U06A9>
-<U06AD> <ar_ng>;<BAS>;<MIN>;IGNORE
-reorder-after <U0648>
-<U06C7> <ar_u>;<BAS>;<MIN>;IGNORE
-<U06C6> <ar_oe>;<BAS>;<MIN>;IGNORE
-<U06C8> <ar_yu>;<BAS>;<MIN>;IGNORE
-<U06CB> <ar_ve>;<BAS>;<MIN>;IGNORE
-<U06D0> <ar_e>;<BAS>;<MIN>;IGNORE
-<U06CC> <ar_i>;<BAS>;<MIN>;IGNORE
-<U064A> <ar_y>;<BAS>;<MIN>;IGNORE
+<ARA_ywha_waw>
+<waw>
+<ARA_ywha_u>
+<ARA_u>
+<ARA_ywha_oe>
+<ARA_oe>
+<ARA_ywha_yu>
+<ARA_yu>
+<ARA_ve>
+<ARA_ywha_e>
+<ARA_e>
+<ARA_ywha_alma>
+<alef_maksura>
+<ARA_yeh>
+<ARA_ywha>
+
+reorder-after   <U0627>
+<ara_h_alef>    <ARA_ywha_alef>;<BAS>;<MIN>;IGNORE
+<U0627>         <alef>;<BAS>;<MIN>;IGNORE
+<ara_h_ae>      <ARA_ywha_ae>;<BAS>;<MIN>;IGNORE
+<U06D5>         <ARA_ae>;<BAS>;<MIN>;IGNORE
+
+reorder-after   <U06AF>
+<U06AD>         <ARA_ng>;<BAS>;<MIN>;IGNORE
+
+reorder-after   <U0646>
+<U06BE>         <ARA_hehd>;<BAS>;<MIN>;IGNORE
+
+reorder-after   <U0648>
+<ara_ywha_waw>  <ARA_ywha_waw>;<BAS>;<MIN>;IGNORE
+<U0648>         <waw>;<BAS>;<MIN>;IGNORE
+<ara_ywha_u>    <ARA_ywha_u>;<BAS>;<MIN>;IGNORE
+<U06C7>         <ARA_u>;<BAS>;<MIN>;IGNORE
+<ara_ywha_oe>   <ARA_ywha_oe>;<BAS>;<MIN>;IGNORE
+<U06C6>         <ARA_oe>;<BAS>;<MIN>;IGNORE
+<ara_ywha_yu>   <ARA_ywha_yu>;<BAS>;<MIN>;IGNORE
+<U06C8>         <ARA_yu>;<BAS>;<MIN>;IGNORE
+<U06CB>         <ARA_ve>;<BAS>;<MIN>;IGNORE
+<ara_ywha_e>    <ARA_ywha_e>;<BAS>;<MIN>;IGNORE
+<U06D0>         <ARA_e>;<BAS>;<MIN>;IGNORE
+<ara_ywha_alma> <ARA_ywha_alma>;<BAS>;<MIN>;IGNORE
+<U0649>         <alef_maksura>;<BAS>;<MIN>;IGNORE
+<U064A>         <ARA_yeh>;<BAS>;<MIN>;IGNORE
+<U0626>         <ARA_ywha>;<BAS>;<MIN>;IGNORE
 
 reorder-end
 
 END LC_COLLATE
 
 LC_CTYPE
-% FIXME
-copy "en_DK"
+copy "i18n"
 END LC_CTYPE
 
 LC_MONETARY
@@ -261,72 +229,84 @@
 END LC_TELEPHONE
 
 LC_TIME
-% FIXME
-abday    "<U0053><U0075><U006E>";"<U004D><U006F><U006E>";/
-         "<U0054><U0075><U0065>";"<U0057><U0065><U0064>";/
-         "<U0054><U0068><U0075>";"<U0046><U0072><U0069>";/
-         "<U0053><U0061><U0074>"
-day      "<U0053><U0075><U006E><U0064><U0061><U0079>";/
-         "<U004D><U006F><U006E><U0064><U0061><U0079>";/
-         "<U0054><U0075><U0065><U0073><U0064><U0061><U0079>";/
-         "<U0057><U0065><U0064><U006E><U0065><U0073><U0064><U0061><U0079>";/
-         "<U0054><U0068><U0075><U0072><U0073><U0064><U0061><U0079>";/
-         "<U0046><U0072><U0069><U0064><U0061><U0079>";/
-         "<U0053><U0061><U0074><U0075><U0072><U0064><U0061><U0079>"
-abmon    "<U004A><U0061><U006E>";"<U0046><U0065><U0062>";/
-         "<U004D><U0061><U0072>";"<U0041><U0070><U0072>";/
-         "<U004D><U0061><U0079>";"<U004A><U0075><U006E>";/
-         "<U004A><U0075><U006C>";"<U0041><U0075><U0067>";/
-         "<U0053><U0065><U0070>";"<U004F><U0063><U0074>";/
-         "<U004E><U006F><U0076>";"<U0044><U0065><U0063>"
-mon      "<U004A><U0061><U006E><U0075><U0061><U0072><U0079>";/
-         "<U0046><U0065><U0062><U0072><U0075><U0061><U0072><U0079>";/
-         "<U004D><U0061><U0072><U0063><U0068>";/
-         "<U0041><U0070><U0072><U0069><U006C>";/
-         "<U004D><U0061><U0079>";/
-         "<U004A><U0075><U006E><U0065>";/
-         "<U004A><U0075><U006C><U0079>";/
-         "<U0041><U0075><U0067><U0075><U0073><U0074>";/
-         "<U0053><U0065><U0070><U0074><U0065><U006D><U0062><U0065><U0072>";/
-         "<U004F><U0063><U0074><U006F><U0062><U0065><U0072>";/
-         "<U004E><U006F><U0076><U0065><U006D><U0062><U0065><U0072>";/
-         "<U0044><U0065><U0063><U0065><U006D><U0062><U0065><U0072>"
-% date formats following ISO 8601-1988
-d_t_fmt  "<U0025><U0059><U002D><U0025><U006D><U002D><U0025><U0064><U0054><U0025><U0054><U0020><U0025><U005A>"
-d_fmt    "<U0025><U0059><U002D><U0025><U006D><U002D><U0025><U0064>"
+
+abday "<U064A>";/
+      "<U062F>";/
+      "<U0633>";/
+      "<U0686>";/
+      "<U067E>";/
+      "<U062C>";/
+      "<U0634>"
+day   "<U064A><U06D5><U0643><U0634><U06D5><U0646><U0628><U06D5>";/
+      "<U062F><U06C8><U0634><U06D5><U0646><U0628><U06D5>";/
+      "<U0633><U06D5><U064A><U0634><U06D5><U0646><U0628><U06D5>";/
+      "<U0686><U0627><U0631><U0634><U06D5><U0646><U0628><U06D5>";/
+      "<U067E><U06D5><U064A><U0634><U06D5><U0646><U0628><U06D5>";/
+      "<U062C><U06C8><U0645><U06D5>";/
+      "<U0634><U06D5><U0646><U0628><U06D5>"
+abmon "<U0642><U06D5><U06BE><U0631><U0649><U062A><U0627><U0646>";/
+      "<U06BE><U06C7><U062A>";/
+      "<U0646><U0648><U0631><U06C7><U0632>";/
+      "<U0626><U06C8><U0645><U0649><U062F>";/
+      "<U0628><U0627><U06BE><U0627><U0631>";/
+      "<U0633><U06D5><U067E><U06D5><U0631>";/
+      "<U0686><U0649><U0644><U0644><U06D5>";/
+      "<U062A><U0648><U0645><U06C7><U0632>";/
+      "<U0645><U0649><U0632><U0627><U0646>";/
+      "<U0626><U0648><U063A><U06C7><U0632>";/
+      "<U0626><U0648><U063A><U0644><U0627><U0642>";/
+      "<U0643><U06C6><U0646><U06D5><U0643>"
+mon   "<U0642><U06D5><U06BE><U0631><U0649><U062A><U0627><U0646>";/
+      "<U06BE><U06C7><U062A>";/
+      "<U0646><U0648><U0631><U06C7><U0632>";/
+      "<U0626><U06C8><U0645><U0649><U062F>";/
+      "<U0628><U0627><U06BE><U0627><U0631>";/
+      "<U0633><U06D5><U067E><U06D5><U0631>";/
+      "<U0686><U0649><U0644><U0644><U06D5>";/
+      "<U062A><U0648><U0645><U06C7><U0632>";/
+      "<U0645><U0649><U0632><U0627><U0646>";/
+      "<U0626><U0648><U063A><U06C7><U0632>";/
+      "<U0626><U0648><U063A><U0644><U0627><U0642>";/
+      "<U0643><U06C6><U0646><U06D5><U0643>"
+
+% %a، %d-%m-%Y، %T (w، dd-mm-YYYY، HH:MM:SS) (قىسقا ھەپتە، 31-12-2012، 23:08:59)
+d_t_fmt  "<U0025><U0061><U060C><U0020><U0025><U0064><U002D><U0025><U006D><U002D><U0025><U0059>/
+<U060C><U0020><U0025><U0054>"
+
+% %a، %d-%m-%Y (w، dd-mm-YYYY) (قىسقا ھەپتە، 31-12-2012)
+d_fmt    "<U0025><U0061><U060C><U0020><U0025><U0064><U002D><U0025><U006D><U002D><U0025><U0059>"
+
+% %T (HH:MM:SS)
 t_fmt    "<U0025><U0054>"
+
 am_pm    "";""
-t_fmt_ampm  ""
-date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
-<U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
-<U0025><U005A><U0020><U0025><U0059>"
+
+% %a، %d-%m-%Y، %T (w، dd-mm-YYYY، HH:MM:SS) (قىسقا ھەپتە، 31-12-2012، 23:08:59)
+date_fmt "<U0025><U0061><U060C><U0020><U0025><U0064><U002D><U0025><U006D><U002D><U0025><U0059>/
+<U060C><U0020><U0025><U0054>"
+
 END LC_TIME
 
 LC_MESSAGES
-% FIXME
-yesexpr  "<U005E><U005B><U0031><U004A><U006A><U0073><U0053><U0079><U0059><U006F><U004F><U005D><U002E><U002A>"
-noexpr   "<U005E><U005B><U0030><U006E><U004E><U005D><U002E><U002A>"
-yesstr      "yes"
-nostr       "no"
+yesexpr "<U005E><U005B><U0079><U0059><U005D><U002E><U002A>"
+noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+yesstr  "<U0059><U0065><U0073>"
+nostr   "<U004E><U006F>"
 END LC_MESSAGES
 
-% Default paper: A4
 LC_PAPER
 height 297
-width 210
+width  210
 END LC_PAPER
 
-% Metric System (ISO 1000)
 LC_MEASUREMENT
 measurement 1
 END LC_MEASUREMENT
 
-LC_NAME
-% FIXME
-copy "en_DK"
+LC_NAME % to be fixed
+copy "zh_CN"
 END LC_NAME
 
-LC_ADDRESS
-% FIXME
-copy "en_DK"
+LC_ADDRESS % to be fixed
+copy "zh_CN"
 END LC_ADDRESS
